pkg/binding/controller.go

```diff
@@ -487,7 +493,7 @@ func (c *Controller) reconcile(ctx context.Context, key util.Key) error {
 		if err != nil {
 			// The resource no longer exist, which means it has been deleted.
 			if errors.IsNotFound(err) {
-				c.logger.Info("object referenced from work queue no longer exists",
+				logger.Info("object referenced from work queue no longer exists",
```

Comment:
Logger message should start with a capital letter like the rest of the changed messages. 

Unrelated to PR: perhaps we want to add some developer-documentation on accepted logging pattern?

Reply:
The one principle you cited is well known, we should not have to write our own copy of it.